### PR TITLE
Fix node source thread exhaustion prevents from detecting down nodes

### PR DESCRIPTION
**Pull request by jrochas**

### CODE REVIEW

**Comment by jrochas** [comment]
retest this please

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java [hunk 1]
```diff
@@ -112,6 +111,10 @@
 
     public static final int EXTERNAL_POOL = 1;
 
+    public static final int PINGER_POOL = 2;
```
Comment (Mykhailenko) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java]: in regular day i would ask for java doc and maybe load "2" from configuration.

**Comment by jrochas** [comment]
retest this please

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java (hunk 1 above)
Comment (jrochas) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java]: Actually the "2" is the index of the thread pools array. But you are right, then the name of the constant should be PINGER_POOL_INDEX.

**Pull request merged**